src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -68,8 +70,11 @@ public without sharing class UTIL_OrgTelemetry_SVC {
         IsEnabled_AddressVerification,
         IsEnabled_AutomaticPayments,
         IsEnabled_CustomizableRollups,
+        IsEnabled_CustomRDSchedules,
```

Comment:
@seanpat09 After discussing this more with @mpusto, we felt that knowing about Custom RD Schedule usage is more of a one-time transient need rather than something we need to follow long term. As such, we'd like to change the `IsEnabled_CustomRDSchedules` boolean to the `DataCount_ONETIME_Use` integer feature parameter (returning 1 for orgs that have CustomRDSchedules in use).

Given that there is still a theoretical 25 feature parameter limit per package, the intent is that the `DataCount_ONETIME_Use` feature parameter will be re-used over time to get telemetry that is on a one-time/short need basis. This will be our first implementation of it.

Reply:
That's a good idea, I'll make the change